carma/launch/carma_src.launch.py

```diff
@@ -116,6 +116,22 @@ def generate_launch_description():
         default_value= 'False',
         description='Flag to enable opening http tunnesl to CARMA Cloud'
     )
+
+    # Declare system_architecture
+    system_architecture = LaunchConfiguration('system_architecture')
+    declare_system_architecture = DeclareLaunchArgument(
+        name = 'system_architecture',
+        default_value = 'single',
+        description = 'Flag to define whether a single compute system or a dual compute system is being used'
+    )
+
+    # Declare host_placement
+    host_placement = LaunchConfiguration('host_placement')
+    declare_host_placement = DeclareLaunchArgument(
+        name = 'host_placement',
+        default_value = 'manager',
+        description = 'Flag to define whether the current active host is a manager or worker for ROS node allocation'
```

Comment:
Same as above, can you define the accepted values for the config in the description?